composer.json

```diff
@@ -16,12 +16,12 @@
         }
     ],
     "require": {
-        "php": "^7.2||^8.0",
-        "doctrine/annotations": "^1.13 || ^2.0",
-        "doctrine/instantiator": "^1.0.3 || ^2.0",
+        "php": "^7.2 || ^8.0",
+        "doctrine/annotations": "^1.14 || ^2.0",
+        "doctrine/instantiator": "^1.3.1 || ^2.0",
         "doctrine/lexer": "^2.0 || ^3.0",
         "jms/metadata": "^2.6",
-        "phpstan/phpdoc-parser": "^0.4 || ^0.5 || ^1.0"
```

Comment:
Sound like a bit big issue - I guess not so many people use it as main dependency.

Reply:
I took a glance at https://packagist.org/packages/phpstan/phpdoc-parser/stats just to see what the impact might be.  [0.4.9's](https://packagist.org/packages/phpstan/phpdoc-parser/stats#0.4.9) a huge outlier and getting a few thousand installs per month on its own (and accounting for most of the 0.x installs in general), but if my math's right, 0.x is only accounting for a little under 3% of all recent installs.  For the 1.x installs, approx. 56% are grabbing the latest minor version, and it looks like approx. 86% are grabbing something matching `^1.20`.  So I don't think this is necessarily as impactful as we might've thought just looking at version ranges.